api/src/lib/github/kubernetes.go

```diff
@@ -0,0 +1,40 @@
+package github
+
+import (
+	"regexp"
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+var portRegex = regexp.MustCompile("servicePort: ([0-9]{4})")
+
+func FetchK8Port(repository RepositoryResponse) ([] string)  {
+	files, err := FetchFolder(repository.Url, "kube/core")
+	if err != nil{
```

Comment:
From my experience with go, anywhere you're going to swallow the `err` (and not report it) you're going to pay a lot of debugging hours.

Reply:
I'll keep that in mind in real production apps 👍